src/winshellext/F3DShellExtension.cxx.in

```diff
@@ -11,8 +11,6 @@
 #include "F3DShellExtensionClassFactory.h"
 #include "RegistryHelpers.h"
 
-#include "F3DShellExtension.h"
```

Comment:
a bit confused why would that not be needed anymore

Reply:
the old .h file was used only to store a vector of the supported extensions. Now the cxx file is configured and we are using the extensions directly as an initializer list.

Reply:
fair enough